homeassistant/components/scene/__init__.py

```diff
@@ -21,11 +21,17 @@
 STATE = 'scening'
 
 CONF_ENTITIES = "entities"
+CONF_NAME = "name"
 
 SCENE_SERVICE_SCHEMA = vol.Schema({
     vol.Optional(ATTR_ENTITY_ID): cv.entity_ids,
 })
 
+PLATFORM_SCHEMA = vol.Schema({
+    vol.Required(CONF_NAME): cv.string,
+    vol.Required(CONF_ENTITIES): cv.ordered_dict(cv.match_all, cv.entity_id),
```

Comment:
I'm not sure about the match_all values here. As opposed to no schema validations I guess it's not that bad, but it could be improved I guess.